api/base/serializers.py

```diff
@@ -13,6 +12,27 @@
 from api.base.exceptions import InvalidQueryStringError, Conflict
 
 
+def format_relationship_links(related_link, self_link, rel_meta, self_meta):
+
+    ret = {
+        'links': {
+            'related': {
+                'href': related_link,
+                'meta': rel_meta
+            },
+            'self': {
+                'href': self_link,
+                'meta': self_meta
+            }
+        }
+    }
+
+    # Not currently returning self links
+    if not ret['links']['self']['href']:
```

Comment:
Once Sam's stuff is merged, if there's no related link at all, there parent dictionary key not be there at all in relationships. That logic may need to be "if there's neither a self nor related link", if we think it would be hypothetically possible to have a self link in a relationship without a related link. If we don't think it's possible, then we don't have to change anything. And if we think the answer will complicate Sam's stuff, then we can defer this discussion until we start using self links.